sharding.go

```diff
@@ -100,6 +100,12 @@ func (s *Sharding) Get(instName string) (endpoints []string) {
 	return s.shards[shardName]
 }
 
+// Get returns a list of endpoints
+func (s *Sharding) GetTwo(instName string) (endpoints []string, endpointsTwo []string) {
+	shardName, shardNameTwo, _ := s.consistInst.GetTwo(instName)
```

Comment:
@VictoriaCCK Does github.com/pkumza/consistent of this version has the method 'GetTwo'?

Reply:
not yet QAQ so I've pulled a merge request of consistent...